server/src/repositories/map.repository.ts

```diff
@@ -134,7 +138,29 @@ export class MapRepository implements IMapRepository {
       this.logger.warn(
         `Response from database for reverse geocoding latitude: ${point.latitude}, longitude: ${point.longitude} was null`,
       );
-      return null;
+
+      const ne_response = await this.naturalEarthCountriesRepository
```

Comment:
We should change the above condition to return if it has a response. Then, de-indent all of this.

Reply:
I updated the code. Definitely makes it clearer!